src/Silk.Core.Logic/Commands/Bot/ShardsCommand.cs

```diff
@@ -41,9 +42,9 @@ public async Task Shards(CommandContext ctx)
             sb.Append('|');
             sb.Append($"{averagePing / shards.Count}".Center("\t[Ping]\t"));
             sb.Append('|');
-            sb.Append($"{Discord.Bot.Instance!.Client.ShardClients.SelectMany(c => c.Value.Guilds.Keys).Count()}".Center("\t[Guilds]\t"));
+            sb.Append($"{shards.SelectMany(c => c.Value.Guilds).Count()}".Center("\t[Guilds]\t"));
```

Comment:
Nit: You can sum over the count of the guilds instead of doing this which produces a new enumerable instead that must be iterated over. Something like `shards.Sum(c => c.Value.Guilds.Count)`